test/integration/targets/postgresql/tasks/postgresql_query.yml

```diff
@@ -360,3 +360,91 @@
     - result.rowcount == 0
     - result.statusmessage == 'VACUUM'
     - result.query_result == {}
+
+#
+# Issue 59955
+#
+- name: postgresql_query - create test table for issue 59955
+  become_user: "{{ pg_user }}"
+  become: yes
+  postgresql_table:
+    login_user: "{{ pg_user }}"
+    login_db: postgres
+    name: test_array_table
+    columns:
+    - arr_col int[]
```

Comment:
It might also be interesting to test this with strings. There's a more that can go wrong with strings.

Reply:
Done, thanks!